### PR TITLE
Improve GraphQL installation instructions

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: _includes/graphql/getting-started.md [hunk 1]
```diff
@@ -21,3 +21,79 @@ After running the CLI command, you should have something like this in your termi
...
+$ cd parse-server
+$ docker build --tag parse-server .
+$ docker run --name my-mongo -d mongo
+$ docker run --name my-parse-server --link my-mongo:mongo -d parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://mongo/test --mountGraphQL --mountPlayground
+```
+
+After starting the server, you can visit [http://localhost:1337/playground](http://localhost:1337/playground) in your browser to start playing with your GraphQL API.
+
+Note:
+* ⚠️ Please do not use `--mountPlayground` option in production as anyone could access your API Playground and read or change your application's data. [Parse Dashboard](#running-parse-dashboard) has a built-in GraphQL Playground and it is the recommended option for production apps.
```
Comment (TomWFox) [inline _includes/graphql/getting-started.md]: I don't think it makes sense to have a bulleted list for one item.

File: _includes/graphql/getting-started.md [hunk 2]
```diff
@@ -21,3 +21,79 @@ After running the CLI command, you should have something like this in your termi
...
+  console.log('REST API running on http://localhost:1337/parse');
+  console.log('GraphQL API running on http://localhost:1337/graphql');
+  console.log('GraphQL Playground running on http://localhost:1337/playground');
+});
+```
+
+After starting the server, you can visit [http://localhost:1337/playground](http://localhost:1337/playground) in your browser to start playing with your GraphQL API.
+
+Note:
+* ⚠️ Please do not mount the GraphQL Playground in production as anyone could access your API Playground and read or change your application's data. [Parse Dashboard](#running-parse-dashboard) has a built-in GraphQL Playground and it is the recommended option for production apps.
```
Comment (TomWFox) [inline _includes/graphql/getting-started.md]: same here

File: _includes/graphql/getting-started.md (hunk 1 above)
Comment (TomWFox) [reply _includes/graphql/getting-started.md]: Screenshot 2019-07-14 at 21 01 40

File: _includes/graphql/getting-started.md [hunk 3]
```diff
@@ -21,3 +21,79 @@ After running the CLI command, you should have something like this in your termi
...
+
+## Running Parse Dashboard
+
+[Parse Dashboard](https://github.com/parse-community/parse-dashboard) is a standalone dashboard for managing your Parse Server apps, including your objects' schema and data, logs, jobs, and push notifications. Parse Dashboard has also a built-in GraphQL Playground that you can use to play with your auto-generated Parse GraphQL API. It is the recommended option for production applications.
```
Comment (TomWFox) [inline _includes/graphql/getting-started.md]:
```suggestion
[Parse Dashboard](https://github.com/parse-community/parse-dashboard) is a standalone dashboard for managing your Parse Server apps, including your objects' schema and data, logs, jobs, and push notifications. Parse Dashboard also has a built-in GraphQL Playground that you can use to play around with your auto-generated Parse GraphQL API. It is the recommended option for production applications.
```

File: _includes/graphql/getting-started.md [hunk 4]
```diff
@@ -21,3 +21,79 @@ After running the CLI command, you should have something like this in your termi
...
+
+The easiest way to run the Parse Dashboard is through its CLI:
+
+```bash
+$ npm install -g parse-dashboard
+$ parse-dashboard --dev --appId APPLICATION_ID --masterKey MASTER_KEY --serverURL "http://localhost:1337/parse" --graphQLServerURL "http://localhost:1337/graphql" --appName MyAppName
+```
+
+Note:
+* To learn more about Parse Dashboard and its setup options, please visit [Parse Dashboard Repository](https://github.com/parse-community/parse-dashboard).
```
Comment (TomWFox) [inline _includes/graphql/getting-started.md]: same here

**Review by TomWFox — CHANGES_REQUESTED** [review]
Looks good, just a couple minor requests

File: _includes/graphql/getting-started.md (hunk 1 above)
Comment (davimacedo) [reply _includes/graphql/getting-started.md]: done

File: _includes/graphql/getting-started.md (hunk 2 above)
Comment (davimacedo) [reply _includes/graphql/getting-started.md]: done

File: _includes/graphql/getting-started.md (hunk 4 above)
Comment (davimacedo) [reply _includes/graphql/getting-started.md]: done

**Review by TomWFox — APPROVED** [review]

**Pull request merged**